lib/log.bash

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+export ERROR_LOG_LEVEL=1
+export WARNING_LOG_LEVEL=2
+export ALL_LOG_LEVEL=3
+
+function _log_debug()
+{
+  _about 'log a debug message by echoing to the screen. needs LOG_LEVEL >= ALL_LOG_LEVEL'
+  _param '1: message to log'
+  _example '$ _log_debug "Loading plugin git..."'
+  _group 'log'
+
+  [[ "$LOG_LEVEL" -ge $ALL_LOG_LEVEL ]] || return
+  echo -e "${echo_green}$1"
```

Comment:
Also: Do we want to include the log level in the message, e.g. like this?

```bash
echo -e "DEBUG: ${echo_green}$1${echo_normal}"
```

The log levels should be aligned in that case, so that the colon is at the same location:

```bash
DEBUG:
 WARN:
ERROR:
```

Reply:
nice idea!
done